android/java/org/chromium/chrome/browser/crypto_wallet/activities/BuySendSwapActivity.java

```diff
@@ -773,6 +776,13 @@ private void adjustControls() {
                 TabUtils.openUrlInNewTab(false, Utils.DEX_AGGREGATOR_URL);
                 TabUtils.bringChromeTabbedActivityToTheTop(this);
             });
+            if (mErcTokenRegistry != null && mCustomAccountAdapter != null) {
+                mErcTokenRegistry.getTokenBySymbol("BAT", token -> {
+                            if (token != null) {
```

Comment:
why do we need that twice? I think we have to delete that one and leave it only in `finishNativeInitialization`

Reply:
It won't work if the view is not populated, `findViewById` will return null.